pkg/reconciler/pipelinerun/tracing.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pipelinerun
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	// Name of the tracer
+	Tracer = "PipelineRunReconciler"
+	// Name of the Annotation used for propogating SpanContext
+	SpanContextAnnotation = "tekton.dev/pipelinerun-span-context"
```

Comment:
We don't have a clear convention for names here, but I think the majority are in camelCase, so I think `pipelineRunSpanContext` would be good.